main.go

```diff
@@ -130,8 +130,18 @@ func (p *gitHubPki) getTeamUsers() (err error) {
 	for _, team := range teams {
 		for _, t := range p.Config.Teams {
 			if *team.Name == t {
-				ghUsers, _, err := p.Client.Teams.ListTeamMembers(context.Background(), *team.ID, nil)
-				checkErr(err, "Failed to list team members for team "+*team.Name+": %v")
+				var ghUsers []*github.User
+				opt := &github.TeamListTeamMembersOptions{
+					ListOptions: github.ListOptions{},
+				}
+				for {
+					pageUsers, resp, err := p.Client.Teams.ListTeamMembers(context.Background(), *team.ID, opt)
```

Comment:
I think `opt` could remain `nil` as it was before since we're not overriding anything.

Reply:
We have to modify the `Page` when of the `opt` object for the calls after the first page.